name/folder.go

```diff
@@ -3,34 +3,39 @@ package name
 import (
 	"regexp"
 	"strings"
+
+	"github.com/gobuffalo/flect"
 )
 
-var alphanum = regexp.MustCompile(`[^a-zA-Z0-9_\-\/]+`)
+var alphanum = regexp.MustCompile(`[^a-zA-Z0-9_]+`)
 
-// Folder creates a suitable folder name
-//	admin/widget = admin/widget
-//	foo_bar = foo_bar
-//	U$ser = u_ser
+// Folder returns a suitable folder name. It removes any special characters
+// from the given string `s` and returns a string consists of alpha-numeric
+// characters.
+//	admin/widget --> admin/widget
+//	adminUser --> admin_user
+//	foo_bar --> foo_bar
+//	Admin --> admin
+//	U$ser --> u_ser
 func Folder(s string, exts ...string) string {
 	return New(s).Folder(exts...).String()
 }
 
-// Folder creates a suitable folder name
-//	admin/widget = admin/widget
-//	foo_bar = foo/bar
-//	U$ser = u/ser
+// Folder returns a suitable folder name. It removes any special characters
+// from the given string `s` and returns a string consists of alpha-numeric
+// characters.
+//	admin/widget --> admin/widget
+//	adminUser --> admin_user
+//	foo_bar --> foo_bar
+//	Admin --> admin
+//	U$ser --> u_ser
 func (i Ident) Folder(exts ...string) Ident {
 	var parts []string
 
-	s := i.Original
-	if i.Pascalize().String() == s {
-		s = i.Underscore().String()
-		s = strings.Replace(s, "_", "/", -1)
```

Comment:
underscores (added by `Underscore` for the word boundaries) were replaced with slash only when the original is a pascal case, but the underscores on the others were not affected